test/XRP/xrp-client-integration.test.ts

```diff
@@ -229,4 +229,93 @@ describe('XrpClient Integration Tests', function (): void {
       AccountRootFlags.checkFlag(AccountRootFlags.LSF_DEPOSIT_AUTH, flags!),
     )
   })
+
+  it('Enable Deposit Auth - sending by unauthorized account fails after enabled', async function (): Promise<
```

Comment:
@amiecorso added this test here - wanted to keep the authorize test clean and seemed like it would be good to have this behavior tested explicitly. lmk what you think!

Reply:
nice, yeah I'm surprised I didn't add this in the first place lol, it's like a basic case.  I'm satisfied that this bolsters the logic in `authorizeSendingAccount` tests.